js/cards.js

```diff
@@ -85,15 +96,41 @@ export async function defineCardComponent() {
       return `${CLOUDINARY_URL}/q_auto/${this.data["Rarity Folder"]}/${this.data["Filename"]}.png`;
     }
 
+    resetCard() {
+      card_count = 10;
+      this.back.classList.remove("hidden");
+      this.front.tabIndex = "-1";
+
+      addHover();
+    }
+
     flipCard() {
-      this.front.classList.toggle("hidden");
-      this.back.classList.toggle("hidden");
+
+      if(gacha_display_selection !== "gacha-grid" && previous_card_front !== undefined){
+        previous_card_front.removeEventListener("mouseover", addAnimation);
+        previous_card_front.removeEventListener("click", addAnimation);
+      }
+
+      this.classList.add("clicked");
+      this.holder.classList.add("flip");
+      previous_card = this;
+      previous_card_front = this.image;
+      card_count++;
+
+      this.addEventListener("animationstart", updateZIndex);
+
+      this.image.classList.add("animated");
+
     }
 
     //Binds the card's onclick events to flip and show the description popup.
     setupOnClickEvents() {
       this.back.onclick = (event) => this.flipCard();
-      this.front.onclick = (event) => this.showDetails();
+      if(CARD_ART_HIDDEN_ON_LOAD){
+
+      } else {
+        this.front.onclick = (event) => this.showDetails();
+      }
```

Comment:
condense to `if(!CARD_ART_HIDDEN_ON_LOAD)`?

Reply:
Makes perfect sense, d'oh. :) Have updated this as suggested.